store/migration/10001__init_schema.sql

```diff
@@ -840,13 +840,18 @@ CREATE TABLE task (
         )
     ),
     `type` TEXT NOT NULL CHECK (`type` LIKE 'bb.task.%'),
-    payload TEXT NOT NULL DEFAULT ''
+    payload TEXT NOT NULL DEFAULT '',
+    `not_before_ts` BIGINT NOT NULL DEFAULT 0
 );
 
 CREATE INDEX idx_task_pipeline_id_stage_id ON task(pipeline_id, stage_id);
 
 CREATE INDEX idx_task_status ON task(`status`);
 
+CREATE INDEX idx_task_not_before_ts ON task(not_before_ts);
+
+CREATE INDEX idx_task_status_not_before_ts ON task(`status`, not_before_ts);
```

Comment:
This is not needed since we have both status and not_before_ts

Reply:
done